react/features/base/conference/middleware.js

```diff
@@ -324,10 +329,17 @@ function _connectionFailed({ dispatch, getState }, next, action) {
  * @private
  * @returns {Object} The value returned by {@code next(action)}.
  */
-function _conferenceSubjectChanged({ getState }, next, action) {
+function _conferenceSubjectChanged({ dispatch, getState }, next, action) {
     const result = next(action);
     const { subject } = getState()['features/base/conference'];
 
+    if (subject) {
+        dispatch({
+            type: SET_PENDING_SUBJECT_CHANGE,
+            subject: undefined
```

Comment:
Your answer to my previous question may answer this as well, but why undefined if we check for subject previously?

Reply:
this is the pending one, so after setting it, there is no pending subject anymore